CMakeLists.txt

```diff
@@ -85,6 +85,9 @@ SET_PROPERTY (TARGET binlogevents PROPERTY IMPORTED_LOCATION ${MYSQL_BIN}/libbin
 ADD_DEFINITIONS (-DDBUG_OFF)
 SET (LINK_TYPE STATIC)
 SET (MYSQL_LIBS mysqlclient binlogevents -lssl -lcrypto)
+if(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
```

Comment:
Just notice: this check is not necessary. `${CMAKE_DL_LIBS}` contains `-ldl` on platforms, where dlopen() / dlclose() require linking with libdl.so.